BuildResidentialHPXML/measure.rb

```diff
@@ -2810,8 +2829,19 @@ def self.set_cooling_systems(hpxml, runner, args)
 
     if [HPXML::HVACTypeCentralAirConditioner].include? cooling_system_type
       hpxml.cooling_systems[-1].cooling_efficiency_seer = args[:cooling_system_cooling_efficiency]
+      if args[:cooling_system_cooling_efficiency] <= 15.0
+        hpxml.cooling_systems[-1].compressor_type = 'single stage'
+        hpxml.cooling_systems[-1].cooling_shr = 0.73
+      elsif args[:cooling_system_cooling_efficiency] <= 21.0
+        hpxml.cooling_systems[-1].compressor_type = 'two stage'
+        hpxml.cooling_systems[-1].cooling_shr = 0.73
+      else
+        hpxml.cooling_systems[-1].compressor_type = 'variable speed'
+        hpxml.cooling_systems[-1].cooling_shr = 0.78
+      end
     elsif [HPXML::HVACTypeRoomAirConditioner].include? cooling_system_type
       hpxml.cooling_systems[-1].cooling_efficiency_eer = args[:cooling_system_cooling_efficiency]
+      hpxml.cooling_systems[-1].cooling_shr = 0.65
```

Comment:
No, don't just copy the default logic from the HPXMLtoOS measure. Either 1) expose them as inputs, or 2) if you don't want to add inputs, remove them from your generated HPXMLs (which means they use the default logic in the HPXMLtoOS measure) and exclude them from your test comparisons.

Reply:
I decided to not expose `cooling_shr` and instead exclude it from the test comparisons.

Reply:
Still need to do something with compressor type.

Reply:
It follows the efficiency value, an exposed argument. So is it actually "hardcoded"?

Reply:
Or maybe you're implying that it's possible, e.g., an efficiency of 16 is entered but the compressor type is `single stage`?